src/bin/dual-iir.rs

```diff
@@ -243,21 +254,25 @@ mod app {
             generator,
         };
 
-        // Spawn a settings update for default settings.
-        settings_update::spawn().unwrap();
-        telemetry::spawn().unwrap();
-        ethernet_link::spawn().unwrap();
-
         // Enable ADC/DAC events
         local.adcs.0.start();
         local.adcs.1.start();
         local.dacs.0.start();
         local.dacs.1.start();
 
-        // Start sampling ADCs.
-        stabilizer.adc_dac_timer.start();
+        // Spawn a settings update for default settings.
+        settings_update::spawn().unwrap();
+        telemetry::spawn().unwrap();
+        ethernet_link::spawn().unwrap();
+        start::spawn_after(100.millis()).unwrap();
```

Comment:
Any reason why we need to start the ADC sampling timer so late? Is it the `init::Monotonics` construction? If so, we can perform that call before starting the timer. I dislike having this additional complexity.

Reply:
That and a lot of other things I tried all didn't help. I assume it's some unfortunate sideffect of the other tasks being scheduled all at once here and the copious `interrupt::free` that the timer queue does, and maybe cold caches plus the increased timer ISR frequency as well. I couldn't figure it out immediately without instrumenting and tracing the entire init to idle transition.
A nice project would be to try `rtic-scope` on this. That would provide a lot of insight into the underlying timing of the various timers and delays in that area.